tools/cldr/cldr-to-icu/build-icu-data.xml

```diff
@@ -283,6 +294,19 @@
                  data than "yue", so this alias is not just rewriting the base language. -->
             <!-- TODO: Find out and document this properly. -->
             <forcedAlias dir="rbnf" source="zh_Hant_HK" target="yue"/>
+
+            <!-- The following elements configure alternate values for some special case paths.
+                 The target path will only be replaced if both it, and the source path, exist in
+                 the CLDR data (new paths will not be added if only the source path exists).
+
+                 Since the paths must represent the same semantic type of data, they must be in the
+                 same "namespace" (same element names) and must not contain value attributes. Thus
+                 they can only differ by distinguishing attributes (either added or modified).
+
+                 This feature is typically used to select alternate translations (e.g. short forms)
+                 for certain paths. -->
+            <!-- <altPath target="//path/to/value[@attr='foo']" source="//path/to/value[@attr='bar']"/> -->
```

Comment:
No actual need for this by default, but the comment + example should be enough for people to work with.